src/modules/CVs/components/details/details.tsx

```diff
@@ -1,7 +1,88 @@
-import CvDetailsForm from "@/modules/cvs/components/cv-details-form/cv-details-form";
+import { useAuthUser } from "@/services/auth-service";
+import { Navigate, useParams } from "react-router-dom";
+import { routes } from "@/router";
+import { useTranslation } from "react-i18next";
+import { useCvById, useCvUpdate } from "@/modules/CVs/api";
+import {
+    CvDetailsSchemaType,
+    getCvDetailsSchema,
+} from "src/modules/CVs/components/details/schemas";
+import { Form } from "@/components/form";
+import { FormTextField } from "@/components/form-text-field";
+import { FormTextarea } from "@/components/form-textarea";
+import { FormSubmitButton } from "@/components/form-submit-button";
+import { Flex, Spin } from "antd";
+import styles from "@/modules/CVs/components/details/details.module.css";
 
 const Details = () => {
-    return <CvDetailsForm />;
+    const currentUser = useAuthUser();
+    const { cvId } = useParams<{ cvId: string }>();
+    if (!currentUser || !cvId) {
```

Comment:
This early return can cause conditional hook usage (useXXX hooks bellow), shouldn't be like that